iConverter/Pages/Main/MainViewController.swift

```diff
@@ -21,14 +21,17 @@ class MainViewController: BaseViewController {
     // MARK: - View Model
     var viewModel: MainViewModel!
     private var interactor: MainInteractor!
+    private var router: MainRouter!
```

Comment:
Normally, we put it in Presenter but since an event needs to be observed from presented screen, this is put here